en/docs/deploy/security/encrypting-passwords-with-cipher-tool.md

```diff
@@ -0,0 +1,93 @@
+# Encrypt Passwords with the Cipher Tool
+
+
+The instructions on this page explain how plain text passwords in configuration files can be encrypted using the secure vault implementation that is built into WSO2 Identity Server.  
+
+!!! tip "Before you begin"
+    - If you are using Windows, you need to have **Ant** (<http://ant.apache.org/>) installed before using the Cipher Tool.
+    - If required, you can enable [single key encryption](../../../deploy/security/using-symmetric-encryption) instead of (the default) asymmetric encryption.
```

Comment:
```suggestion
    - If required, you can enable [single key encryption](../../../deploy/security/use-symmetric-encryption) instead of (the default) asymmetric encryption.
```

Reply:
This step is actually not needed since symmetric encryption is now enabled by default from 5.11.0 onwards